docling_core/types/doc/labels.py

```diff
@@ -28,12 +28,22 @@ class DocItemLabel(str, Enum):
     # Additional labels for markup-based formats (e.g. HTML, Word)
     PARAGRAPH = "paragraph"  # explicitly a paragraph and not arbitrary text
     REFERENCE = "reference"
+    INVISIBLE_TEXT = "invisible_text"
 
     def __str__(self):
         """Get string value."""
         return str(self.value)
 
 
+class InvisibleTextLabel(str, Enum):
```

Comment:
I would not introduce an additional dimension just for`InvisibleTextLabel`, let's avoid to have a second-level dependency to `DocItemLabel.INVISIBLE_TEXT`, and just put straight:

```
DocItemLabel.COMMENT
DocItemLabel.NOTE
```

as regular labels. Then, we can add these to the furniture and enable that the furniture can be iterated over together with the rest.